.github/workflows/ci-policies.yml

```diff
@@ -10,10 +10,10 @@ jobs:
         run: |
           echo "Your head ref is ${{ github.head_ref }}."
           echo "Your base ref is ${{ github.base_ref }}."
-      - name: Fail if try to push release from non-master branch
-        if: ((github.base_ref == 'beta-release' || github.base_ref == 'release') && github.head_ref != 'master') && (github.event.pull_request.user.login != 'noschiff')
```

Comment:
Does removing `github.event.pull_request.user.login != 'noschiff'` mean that anyone can merge the pull request into release now?

Reply:
I don't have any opinions on it, was just wondering what removing this line does.

Reply:
Removing this actually makes the check more conservative; that is, there are no cases where the failure would trigger before but not now.